classes/submission/sectionEditor/SectionEditorSubmissionDAO.inc.php

```diff
@@ -833,10 +833,8 @@ function &getReviewersForArticle($journalId, $articleId, $round, $searchType = n
 				(SELECT MAX(ac.date_notified) FROM review_assignments ac WHERE ac.reviewer_id = u.user_id AND ac.date_completed IS NOT NULL) AS latest,
 				(SELECT AVG(ac.date_completed-ac.date_notified) FROM review_assignments ac WHERE ac.reviewer_id = u.user_id AND ac.date_completed IS NOT NULL AND ac.cancelled = 0) AS average
 			 FROM users u
-				LEFT JOIN review_assignments ra ON (ra.reviewer_id = u.user_id)
```

Comment:
This seems to be used as well, s. https://github.com/pkp/ojs/blob/ojs-dev-2_4/classes/submission/sectionEditor/SectionEditorSubmissionDAO.inc.php#L823, right?